src/script/pages/app-home.ts

```diff
@@ -288,16 +288,22 @@ export class AppHome extends LitElement {
         console.error('Error getting site', err.message);
 
         this.errorGettingURL = true;
+
+        // couldnt get manifest, thats ok
+        // lets continue forward with the default
+        // zeroed out results. 
+        const goodURL = getURL();
+        Router.go(`/testing?site=${goodURL}`);
       }
 
       this.gettingManifest = false;
     }
   }
 
-  updateProgress(progress: ProgressList) {
-    progress.progress[0].items[0].done = Status.DONE;
+  updateProgress(progressData) {
```

Comment:
add interface

Reply:
Ahhhhhhh, I think i messed this up when I merged some changes, good find